### PR TITLE
Add Student Form

### PR DESCRIPTION
**Pull request by contact-tarun**
Student Form Complate, and save server side. next task , show all student data in student Update table,

### CODE REVIEW

**Review by GauravKalbhor — APPROVED** [review]
Ok done next task to perform crud operation in student saction

File: CMS/app/static/css/admin_dashboard.css [hunk 1]
```diff
@@ -5,7 +5,7 @@ section {
   background-color: rgb(201, 201, 201);
   display: grid;
   grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
-  grid-template-rows: 0.4fr 0.3fr 0.5fr 1fr 1.7fr 1fr 1fr 1fr;
+  grid-template-rows: 0.4fr 0.3fr 0.1fr 0.3fr 1.7fr 1fr 1fr 1fr;
```
Comment (contact-tarun) [inline CMS/app/static/css/admin_dashboard.css:8]: done

File: CMS/app/static/css/admin_dashboard.css [hunk 2]
```diff
@@ -5,7 +5,7 @@ section {
   background-color: rgb(201, 201, 201);
   display: grid;
   grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
-  grid-template-rows: 0.4fr 0.3fr 0.5fr 1fr 1.7fr 1fr 1fr 1fr;
```
Comment (contact-tarun) [inline CMS/app/static/css/admin_dashboard.css:8]: delet

File: CMS/app/static/css/admin_dashboard.css [hunk 3]
```diff
@@ -166,4 +166,4 @@ section {
 .content4-attendence div,
 h1 {
   text-align: center;
-}
```
Comment (contact-tarun) [inline CMS/app/static/css/admin_dashboard.css:169]: delet

File: CMS/app/templates/admin/admin_dashboard.html [hunk 4]
```diff
@@ -53,7 +54,7 @@
           <i class="fa fa-caret-down"></i>
         </button>
         <div class="dropdown-container">
-          <a href="#">Student Add</a>
```
Comment (contact-tarun) [inline CMS/app/templates/admin/admin_dashboard.html:56]: delet

File: CMS/app/templates/admin/admin_dashboard.html [hunk 5]
```diff
@@ -99,7 +101,13 @@ <h1>59000</h1>
         <div>Total Course</div>
         <h1>12</h1>
       </div>
-      <div class="content8"></div>
+      {% block content %}
```
Comment (contact-tarun) [inline CMS/app/templates/admin/admin_dashboard.html:104]: delet

File: CMS/app/urls.py [hunk 6]
```diff
@@ -1,7 +1,9 @@
-
```
Comment (contact-tarun) [inline CMS/app/urls.py:1]: delet

File: CMS/app/urls.py [hunk 7]
```diff
@@ -1,7 +1,9 @@
...
-
```
Comment (contact-tarun) [inline CMS/app/urls.py:2]: delet

File: CMS/app/urls.py [hunk 8]
```diff
@@ -1,7 +1,9 @@
...
 from django.urls import path
 from .views import *
 urlpatterns = [
     path("",home,name='home'),
-]
```
Comment (contact-tarun) [inline CMS/app/urls.py:7]: delet

File: CMS/app/views.py [hunk 9]
```diff
@@ -1,7 +1,61 @@
 from django.shortcuts import render
 from django.http import  HttpResponse
+from .models import *
+from .forms import *
 
 # Create your views here.
 
+
 def home(request):
-    return render (request,'app/home.html')
```
Comment (contact-tarun) [inline CMS/app/views.py:7]: delet

**Review by GauravKalbhor — APPROVED** [review]

**Review by GauravKalbhor — APPROVED** [review]

**Pull request merged**